cmd/nodecmd/refreshIPs.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package nodecmd
+
+import (
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/spf13/cobra"
+)
+
+func newRefreshIPsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "refresh-ips [clusterName]",
+		Short: "(ALPHA Warning) Refresh IPs for nodes with dynamic IPs in the cluster",
+		Long: `(ALPHA Warning) This command is currently in experimental mode.
+
+The node refresh-ips command obtains the current IP for all nodes with dynamic IPs in the cluster,
+and updates the local node information used by CLI commands.`,
+		SilenceUsage: true,
+		Args:         cobra.ExactArgs(1),
+		RunE:         refreshIPs,
+	}
+
+	cmd.Flags().StringVar(&awsProfile, "aws-profile", constants.AWSDefaultCredential, "aws profile to use")
```

Comment:
should we ask for clustername and  aws profile if it's not provided ?

Reply:
clusterName is the positional argument, we usually fail for that one
regarding awsProfile, we usually defer it to flag for the other commands